### PR TITLE
Added "configureBatchActions" method to AdminExtensionInterface

### PR DESCRIPTION
**Pull request by core23**
~~Please schedule for 3.0~~
Please schedule for 4.0

Refs https://github.com/sonata-project/SonataAdminBundle/pull/3548

### CODE REVIEW

**Comment by OskarStark** [comment]
done, thanks :+1:

**Comment by core23** [comment]
Added missing Changelog and Upgrade notes

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Add the ``configureActionButtons`` method to the AdminInterface
 - Add the ``configureActionButtons`` method to the AdminExtensionInterface
+- Add the ``configureBatchActions`` method to the AdminExtensionInterface
```
Comment (OskarStark) [inline CHANGELOG.md]: not to the AdminInterface?

Comment (core23) [inline CHANGELOG.md]: Nope, the `AdminInterface` doesn't have this method.

**Comment by Soullivaneuh** [comment]
@core23: Thank you. If you think documentation is missing, feel free to add it on 3.x. :+1:

**Pull request merged**